### DISCUSSION TITLE
Wat ga je doen vandaag? -19 juni

### DISCUSSION THREAD

**Discussion by Sensinki** [Sensinki/Blok-Tech-Team-4A#22] [Wat ga je doen vandaag?]
Ik ga met de MVC model werken, als ik moe wordt van, dan ga ik terug naar de liked chats switchen

**Comment by D0nnaz** [comment]
CSS en JS is nu minified en geautomatiseerd met Grunt. Dus ik ga zo een pull request doen, had nog een vraag. Wil even zeker weten of jij de laatste versie daarvoor nog hebt. Stel er gaat iets mis dan is er niks aan de hand :) Is dat het geval? Verder ik ga aan de WIKI werken en misschien het MVC model.
